lua/octo/reviews/init.lua

```diff
@@ -51,6 +51,36 @@ function Review:create(callback)
   }
 end
 
+-- Get review threads without start a review.
+function Review:populate_threads(callback)
+  local query =
+    graphql("review_threads_query", self.pull_request.owner, self.pull_request.name, self.pull_request.number)
+  -- print query to file /tmp/query
+  local file = io.open("/tmp/query", "w")
+  file:write(query)
+  file:close()
```

Comment:
Is this just for testing?

Reply:
Yup 🤦 

forgot about that. Ill remove.